### PR TITLE
Adapt to changes of MemberData in CMFCore

### PR DESCRIPTION
**Pull request by pbauer**
Do not merge! 
Fixes https://github.com/plone/Products.PlonePAS/issues/24 for compatability with Zope 4

### CODE REVIEW

File: src/Products/PlonePAS/tools/memberdata.py [hunk 1]
```diff
@@ -238,11 +223,17 @@ def _getPlugins(self):
 InitializeClass(MemberDataTool)
 
 
-@implementer(IManageCapabilities)
-class MemberData(BaseMemberData):
+# FIXME: MemberData should probably be a more specific adapter
+# instead of a subclass.
```
Comment (jensens) [inline src/Products/PlonePAS/tools/memberdata.py]: If the Interface is the same, there is no need to subclass IMember just for the sake of being more specific, so IMO this is perfectly fine.

Comment (pbauer) [reply src/Products/PlonePAS/tools/memberdata.py]: Thanks for the review. I'll remove the the FIXME.

**Comment by pbauer** [comment]
I guess we need a legacy-branch when we merge this. Because this will not work in Plone 5.x. Merging can wait untill we have a stable 5.1 release and/or a new 6.0 branch for the coredev.

**Comment by jensens** [comment]
Imo: Branch the current master as 5.x, and set it in coredev sources.cfg as the current. Increase major version number here to 6.0.dev0, in setup.py depend on CMFCore >=... and go on.

**Pull request merged**